src/sentry/api/endpoints/project_user_reports.py

```diff
@@ -106,9 +120,19 @@ def post(self, request, project):
             report.event_user_id = euser.id
 
         try:
-            report.group = Group.objects.from_event_id(project, report.event_id)
-        except Group.DoesNotExist:
-            pass
+            event = Event.objects.filter(project_id=project.id,
```

Comment:
@mattrobenolt Good catch with the index. I didn't notice that initially

Reply:
It's not that it's just missing the index, it's more about the uniqueness. A single `event_id` is not globally unique, it's only unique within a single project. So by not scoping it to project as well, it might return the wrong data.